go/client/ctclient/ctclient.go

```diff
@@ -30,7 +31,7 @@ func signatureToString(signed *ct.DigitallySigned) string {
 }
 
 func getSTH(logClient *client.LogClient) {
-	sth, err := logClient.GetSTH()
+	sth, err := logClient.GetSTH(context.Background())
```

Comment:
nit: pass ctx to getSTH and have context.Background() used once only in the main().